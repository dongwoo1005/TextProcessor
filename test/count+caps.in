stdin count l allcaps
Cs246 Hello World!
