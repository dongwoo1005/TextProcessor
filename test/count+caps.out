CS246
HE12O
WOR3D!
